internal/services/eventgrid/eventgrid_partner_registration_test.go

```diff
@@ -0,0 +1,118 @@
+package eventgrid_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/eventgrid/2022-06-15/partnerregistrations"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type EventgridPartnerRegistrationTestResource struct{}
+
+func TestAccEventGridPartnerRegistration_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_eventgrid_partner_registration", "test")
+	r := EventgridPartnerRegistrationTestResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data, ""),
```

Comment:
Can we make the test config's not pass parameters?
```suggestion
			Config: r.basic(data),
```